vllm/_custom_ops.py

```diff
@@ -154,10 +154,15 @@ def marlin_gemm(a: torch.Tensor, b_q_weight: torch.Tensor,
 
 
 # fp8
-def scaled_fp8_quant(input: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
-    scale = torch.zeros(1, device=input.device, dtype=torch.float32)
+def static_scaled_fp8_quant(input: torch.Tensor, scale: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
```

Comment:
Is it possible to unify this op like the following?
```python
def scaled_fp8_quant(input, scale=None):
    output = torch.empty_like(input, dtype=torch.float8_e4m3fn)
    if scale is None:
        scale = torch.zeros(1, device=input.device, dtype=torch.float32)
        vllm_ops.dynamic_scaled_fp8_quant(output, input, scale)
    else:
        vllm_ops.static_scaled_fp8_quant(output, input, scale)
    return output, scale
```

Reply:
That's what I first had and then cuda graph capture was complaining at the `if scale is None:` place with "operation not allowed" hmmm (I would prefer it that way if we can) -- that was the "fix cuda graph" commit haha

Reply:
ahh good point...then never mind. We just need a comment to prevent others from suggesting this change again lol

Reply:
Interesting, I had checked `if scale:`. If I do your `if scale is None:` it works. I'm changing it :)